opengrok-indexer/src/test/resources/analysis/typescript/sample.ts

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2019 GitHub
```

Comment:
hmm, below is MIT ? if yes I am OK

Reply:
Yes. I scrounge for substantial samples that are CDDL-compatible